app/frontend/src/components/admin/AdminPage.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <v-tabs>
+  <v-tabs :class="{ 'dir-rtl': isRTL }">
```

Comment:
There is a small problem with the tabs: when you change languages, the line under the current tab is not redrawn. For example, if you go from English (FORMS) to a language with a longer word, like Portuguese (FORMULARIOS), the line isn't long enough.

![image](https://github.com/bcgov/common-hosted-form-service/assets/35532993/15721d71-9eb0-4fd1-aeb2-2da9e78d5842)

This isn't such a big deal, but if you switch to a RtoL language, the line is on the wrong side of the screen.

![image](https://github.com/bcgov/common-hosted-form-service/assets/35532993/2b8f6826-58ec-4e40-a7c9-35c77319d888)

If you slightly resize the browser window to cause a redraw, then it is fine for either case.

![image](https://github.com/bcgov/common-hosted-form-service/assets/35532993/ed5aa069-4d54-473b-8616-b60bce60424f)


Can we force a redraw of the v-tab when the language changes?

Reply:
We should be able to do that. I will work on it

Reply:
![image](https://github.com/bcgov/common-hosted-form-service/assets/101672465/85782b28-d939-47de-ba83-43f65f9903d0)

It's still doing this one, but the line is moving to the right spot when switching to RtoL.

Reply:
Vuetify uses the left position, and there is a way they calculate that which is not exposed. I can create new ticket. What do you think?

Reply:
I think it's fine, this is only visible in the admin page